### ISSUE TITLE
Find and fix accessibility issues

### ISSUE THREAD

**Issue by hortongn** [uclibs/scholar_uc#1267]
Use the aXe Chrome plugin to do an initial check for accessibility.

After Scholar has been deployed to QA, work with the UC QA folks to run an official scan on Scholar.  Create new issues for each type of accessibility violation found.

If the UC switches to a new accessibility scanner, run it against Scholar as well.

**Comment by flutterflies** [comment]
All of the issues from our scan on 9/18 that appear to be non issues.

#### Contrast issues
[Search for User](https://wsuofc.dequecloud.com/worldspace/issue/edit/3442313) - This is coming up as a potential violation because the contrast cannot be determined due to the background being a gradient. The gradient is, according to the Chrome dev tools, from `#EEEEEE` to `#FFFFFF` with a text color of `#444444` which provides a contrast ratio between `8.39` and `9.74` which is well within the WCAG AA standard minimum of `4.5`

[Contact the team](https://wsuofc.dequecloud.com/worldspace/issue/edit/3045412) - This has come up as an issue on all three scans but it appears to be a non issue. However the contrast ratio of `5.0` (background: `#FFFFFF`, foreground: `#E00122`) is WCAG AA compliant. Additionally there are seven other links on the page with the exact same foreground and background colors that are not coming up as violations.

#### Table columns
* [Issue 1](https://wsuofc.dequecloud.com/worldspace/issue/edit/3045562)
* [Issue 2](https://wsuofc.dequecloud.com/worldspace/issue/edit/3044955)
* [Issue 3](https://wsuofc.dequecloud.com/worldspace/issue/edit/3044588)
* [Issue 4](https://wsuofc.dequecloud.com/worldspace/issue/edit/3044418)
* [Issue 5](https://wsuofc.dequecloud.com/worldspace/issue/edit/3044410)
* [Issue 6](https://wsuofc.dequecloud.com/worldspace/issue/edit/3044402)

These six issues are all related to table headers not being associated with their respective columns. In each of these cases every `<th>` element has a `scope='col'` attribute which should be sufficient to associate a table header with it's column.
